### ISSUE TITLE
Nasal Nitric Oxide Measurement

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5816]
**Preferred term label:**
https://www.atsjournals.org/doi/full/10.1513/AnnalsATS.201701-062SR

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Label: Low nasal nitric oxide level
Definition: Reduced level of nasal nitric oxide (nNO). Current American Thoracic Society/European Respiratory Society (ATS/ERS) guidelines for nNO measurements recommend air aspiration via a nasal probe while the subject exhales through the mouth against resistance in order to maintain velum closure. 
 PMID:28481653
Comment: Nasal nitric oxide (nNO) is measurable in nasal air and found to be low in cystic fibrosis (CF) and very low in primary ciliary dyskinesia (PCD) .

**Comment by pnrobinson** [comment]
added term

**Comment by pnrobinson** [comment]
Parent: Functional respiratory abnormality
